src/settings/components/ProxyServerSettingsConfig/ProxyServerSettingsEdit/ProxyServerSettingsEdit.test.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { renderWithIntl, TestForm } from '@folio/stripes-erm-testing';
+import { Button, TextField, TextArea } from '@folio/stripes-testing';
+
+import translationsProperties from '../../../../../test/helpers';
+import { input, meta, mutators, platforms } from './testResources';
+import ProxyServerSettingsEdit from './ProxyServerSettingsEdit';
+
+const onDelete = jest.fn();
+const onSave = jest.fn();
+const onCancel = jest.fn();
+const onSubmit = jest.fn();
+
+describe('ProxyServerSettingsEdit', () => {
+  let renderComponent;
+  beforeEach(() => {
+    renderComponent = renderWithIntl(
+      <TestForm onSubmit={onSubmit}>
+        <ProxyServerSettingsEdit
+          input={input}
```

Comment:
We should never send `input` or `meta` props in directly unless there's a very good reason to.